src/main/kotlin/io/emeraldpay/dshackle/upstream/calls/DefaultEthereumMethods.kt

```diff
@@ -196,7 +197,7 @@ class DefaultEthereumMethods(
             possibleNotIndexedMethods.contains(method) -> NotNullQuorum()
             specialMethods.contains(method) -> {
                 when (method) {
-                    "eth_getTransactionCount" -> AlwaysQuorum()
+                    "eth_getTransactionCount" -> MaximumValueQuorum()
                     "eth_getBalance" -> AlwaysQuorum()
                     "eth_blockNumber" -> NotLaggingQuorum(0)
                     else -> AlwaysQuorum()
```

Comment:
Also it's necessary to add again `eth_sendRawTransaction` here with `BroadcastQuorum` cause I removed it due to old logic in `BroadcastReader`